Demos/FluentUIDemo_macOS/FluentUITestViewControllers/TestDatePickerController.swift

```diff
@@ -14,14 +14,12 @@ class TestDatePickerController: NSViewController {
 		let calendar = Calendar.current
 		let date = calendar.date(from: DateComponents(year: 2019, month: 12, day: 9))
 
-		datePickerController = DatePickerController(date: date, calendar: calendar, style: .dateTime)
-		menuDatePickerController = DatePickerController(date: date, calendar: calendar, style: .dateTime)
-
-		datePickerController?.delegate = self
-		menuDatePickerController?.delegate = self
-
-		datePickerController?.hasEdgePadding = true
-		menuDatePickerController?.hasEdgePadding = true
+		DatePickerLocation.allCases.forEach { location in
```

Comment:
Totally ignorable - I've mostly stopped using `.forEach` in favor of old-fashioned `for...in`. I personally pretty strongly agree with https://forums.swift.org/t/do-we-want-foreach/56929/2m, plus it makes debugging easier to just be able to step through a for loop and not to have to set a separate breakpoint inside because it's fired as a closure

Reply:
I think it's still a matter of preference and not a proven guideline so I'm good either way

Reply:
Changed all the complex multi-line loops to use `for x in y`, but left the one-liners as-is